state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -58,6 +58,10 @@ use sp_std::{
 };
 pub use weights::WeightInfo;
 
+/// Max allowed value for the number of blocks to keep retrying a swap before it is refunded
+pub const MAX_RETRY_DURATION_BLOCKS: u32 =
+	3600 / (SECONDS_PER_BLOCK as u32 * SWAP_RETRY_DELAY_BLOCKS);
```

Comment:
This name still seems too generic (we have *lots* of retry for all sorts of things). Why not `max_swap_lifetime` or something.

And I still don't understand the relation to `SWAP_RETRY_DELAY_BLOCKS`. Why would the max lifetime of a swap be lower if the swap retry delay is higher?

> The idea was to tie it to the number of times the swap can be retried.
(from another comment)

Not sure I understand.

Reply:
Would it not make more sense if we define MAX_NUMBER_OF_SWAP_RETRIES and the MAX_LIFETIME = MAX_NUMBER_OF_SWAP_RETRIES * SWAP_RETRY_DELAY ?

Reply:
Renamed `MAX_SWAP_RETRY_DURATION_BLOCKS`, and I no longer (incorrectly) divide by `SWAP_RETRY_DELAY_BLOCKS`. Moved `SWAP_RETRY_DELAY_BLOCKS` out of cf-primitives too since it is not needed there anymore.